src/main/java/seedu/address/logic/parser/BookCommandParser.java

```diff
@@ -28,7 +28,8 @@ public BookCommand parse(String args) throws ParseException {
         }
 
         Appointment appointment = ParserUtil.parseAppointment(argMultimap.getValue(PREFIX_REASON).get(),
-                argMultimap.getValue(PREFIX_DATE).get());
+                argMultimap.getValue(PREFIX_DATE).get(),
+                false);
```

Comment:
Perhaps adding the boolean value here is unneeded as the default value should be false and can be inserted in the return statement of the parseAppointment method instead.